src/main/java/model/SentimentInfo.java

```diff
@@ -0,0 +1,34 @@
+package model;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import org.apache.hadoop.io.Writable;
+
+public class SentimentInfo implements Writable {
+
+  private SentimentType sentimentType;
+  public SentimentInfo() {
+  }
+
+  public SentimentInfo(SentimentType sentimentType) {
+    this.sentimentType = sentimentType;
+  }
+
+  @Override
+  public void write(DataOutput dataOutput) throws IOException {
+    dataOutput.writeUTF(sentimentType.name());
+  }
+
+  @Override
+  public void readFields(DataInput dataInput) throws IOException {
+    sentimentType = SentimentType.valueOf(dataInput.readUTF());
+  }
+
```

Comment:
We might need a getter for retrieving sentiment type object

Reply:
Can we please add it to this branch? so that we keep modifying things seperately.